scripts/release-bit-verification.sh

```diff
@@ -17,6 +17,9 @@ shout() {
 }
 # Check SHASUM for all the binary files and there should be no difference
 
+#For erroring out in case of error
+set -e
+
 # Create a Temp directory
```

Comment:
Regarding error management, can you check that the argument (`$1`) passed to this script is a valid file that exists even before creating the temporary working dir. I think we should not even go further if the path to the RPM file passed is a not valid.

Reply:
Yeah I'll check that

Reply:
I checked this and it looks like its working fine and without any errors

Reply:
Ok I'm looking into that